<ui version="4.0" >
 <class>dialogOptions</class>
 <widget class="QDialog" name="dialogOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Options</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res.qrc" >
    <normaloff>:/Toolbar/pics/ascii-design.png</normaloff>:/Toolbar/pics/ascii-design.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Figlet Path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineFigletPath" />
       </item>
       <item>
        <widget class="QPushButton" name="buttonOpenFiglet" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Figlet fonts path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineFigletFontsPath" />
       </item>
       <item>
        <widget class="QPushButton" name="buttonOpenFigletFonts" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc" />
 </resources>
 <connections/>
</ui>
